autoware_adapi_v1_msgs/vehicle/msg/VehicleDimensions.msg

```diff
@@ -0,0 +1,9 @@
+float32 wheel_radius
+float32 wheel_width
+float32 wheel_base
+float32 wheel_tread
+float32 front_overhang
+float32 rear_overhang
+float32 left_overhang
+float32 right_overhang
+float32 height
```

Comment:
According to [this document](https://autowarefoundation.github.io/autoware-documentation/main/design/autoware-interfaces/components/vehicle-dimensions/), footprint may be defined and used in some cases.
We may have to start discussing whether footprint is mandatory as Vehicle dimensions or not
cc @xmfcx

Reply:
I think it can be mandatory. But we should definitely add it to this message.

Reply:
Add footprint https://github.com/autowarefoundation/autoware_adapi_msgs/pull/28/commits/c27b19a8017990de9c63480b4591917f65f9e673. if not specified, it can be empty or a rectangle made from the other dimensions.